src/server/graphql/mutations/createReflection.js

```diff
@@ -74,6 +74,7 @@ export default {
       content: normalizedContent,
       isActive: true,
       meetingId,
+      phaseItem,
```

Comment:
@mattkrick adding the `phaseItem` object here seems to do the trick, I think this needed to be in the payload that is used to end the meeting and send the summary